app/src/main/res/values/strings.xml

```diff
@@ -43,6 +43,10 @@
     <string name="validating_partner_id" translatable="false">Validating Partner ID</string>
     <string name="shift_progress" translatable="false">Shift Progress</string>
     <string name="pending_registrations">Applications Pending Upload</string>
+
+    <!--Wording here fits the requirement that we don't make a user "worry" about a failed upload-->
+    <string name="failed_registration_uploads">Requires Admin For Upload</string>
```

Comment:
needs `translatable="false"`

Reply:
👍 Should `translatable="false"` be in the strings on either side of this as well?

Reply:
ya, but technically out of scope